index.js

```diff
@@ -0,0 +1,50 @@
+
+// . reduce ()
+// obj
+// we have ; 
+// arrays, properties, objects(string, numbers)
+
+function totalSalaryTest(lineup) {
+  return lineup.reduce((salary, player) => {
+    return salary + player.salary
+  }, 0)
+}
+
+function wholeTeamTest(lineup) {
+  return lineup.reduce((teams, player) => {
+    teams[player.teamId] = teams[player.teamId] === undefined ? 1 : teams[player.teamId] + 1
+
+    return teams
+  }, {})
+}
+
+function gamesTest(lineup) {
+  return lineup.reduce((games, player) => {
+    games[player.gameId] = games[player.gameId] === undefined ? 1 : games[player.gameId] + 1
+
+    return games
+  }, {})
+}
```

Comment:
Same with these, they aren't testing anything they are reformatting the data. Something like `getGamesCounts` would be a better naming scheme.